apps/computeVSD/computeVSD.cpp

```diff
@@ -195,10 +199,41 @@ class ComputeVSD : public CommandLineApplication
 
         VolumePtr output = source->GetOutput();
         output->SetRegions( fivox::FloatVolume::RegionType( vIndex, vSize ));
-        output->SetSpacing( volumeHandler.computeSpacing( ));
 
-        const fivox::AABBf& bbox = vsdLoader->getBoundingBoxSomas();
-        output->SetOrigin( volumeHandler.computeOrigin( bbox.getCenter( )));
+        fivox::AABBf bboxSomas;
+        const auto& somaPositions = vsdLoader->getSomaPositions();
+        for( const auto& position : somaPositions )
+            bboxSomas.merge( position );
+
+        const auto spacing = volumeHandler.computeSpacing();
```

Comment:
Spacing is the distance between one pixel/voxel and the next. Similar to the pixel/voxel size